docs/developer/upgrade-guide/3-0-to-3-1.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Saleor 3.1
+---
+
+## Preparation
+
+The 3.1 version introduces some breaking changes that were necessary to ensure highest quality of our API and security. There were no changes in public APIs, so your storefronts do not need any changes to be compatible.
```

Comment:
> There were no changes in public APIs, so your storefronts do not need any changes to be compatible.

Are you sure of this? I thought that we remove a lot of deprecated fields but also introduced a multichannel functionality.

Reply:
This is a migration guide for 3.1 (from 3.0), if there were any deprecated fields they would already be dropped in 3.0.

Reply:
Oh, sorry. My bad